app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,95 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      erb :'/tweets/index'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/new'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    if logged_in?
+      @tweet = Tweet.create(params)
+
+      if @tweet.save
+        current_user.tweets << @tweet
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.